test/mocha/01-api.js

```diff
@@ -0,0 +1,270 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {agent} = require('bedrock-https-agent');
+const fs = require('fs');
+const {httpClient} = require('@digitalbazaar/http-client');
+const path = require('path');
+
+const {readFile} = fs.promises;
+
+describe('httpClientHandler', () => {
+  const BASE_URL = `https://localhost:18443`;
+  it('configured route should work properly', async () => {
+    let res;
+    let err;
+    try {
+      res = await httpClient.get(`${BASE_URL}/test`, {agent});
+    } catch(e) {
+      err = e;
+    }
+    should.exist(res);
+    should.not.exist(err);
+    res.status.should.equal(200);
+    res.data.success.should.equal(true);
+  });
+  it('should handle error properly', async () => {
```

Comment:
So this test isn't handling an error, it's just receiving one, right? I got confused on the context here so I went to look up the source for `httpClientHandler` ... but that doesn't seem to be a thing? I think the context name for what's being tested here needs to be more clear. Are we testing "configured routes" or something? Once that's updated, I think the test names may also need to be tweaked to better reflect what's going on. My best guess is that we're just testing that configured routes return certain success or error responses.

Reply:
Yep, I'm testing the configured routes. I copied the describe block from somewhere else when setting up the test and forgot to update the description, sorry about the confusion. Will make the changes, thanks.

Reply:
Updated here ac8eb8e